### ISSUE TITLE
Option to stay in hint mode until <escape> is pressed

### ISSUE THREAD

**Issue by oxysoft** [cmcaine/tridactyl#353]
Is there currently a way to stay in hint mode so you can e.g. press `F` and chain open a few links without having to redo the first `F` key every time? If not, then I think it would be a worthwhile addition! It'd be available as either another shortcut, or modifier to the `f` key, maybe you press another key after entering hint mode? It'd have to be anything but a letter since those are used for hints.

**Comment by oxysoft** [comment]
I just thought up an idea: what if instead of adding a new key for a different mode, we pressed the letters with shift held down (i.e. pressing hints in uppercase instead of lowercase) and it would keep you in hint mode after activating a hint? That way, it would work with all other hint modes. (opening images, selecting element pages, etc.)

**Comment by bovine3dom** [comment]
Sorry, I missed this issue. We already touched on it in issue #28 and ages ago started to write it, so there's probably an ancient branch somewhere (we call the mode "rapid hinting").

I think I got stuck because I just wanted a general way of repeating a command until escape was pressed, but couldn't decide on the best way to do that.

Your second suggestion could work OK for letters, but lots of people (dozens of us) use numerals as hint characters for use with the Vimperator style hinting.
